server/services/mlog/mlog.go

```diff
@@ -0,0 +1,256 @@
+// Package mlog provides a simple wrapper around Logr.
```

Comment:
Is this the same found in mattermost-server repo? in that case, can we just import and use the mattermost-server one?

Reply:
This is a very different mlog from mm-server.  The mm-server mlog has over time become a bit of a mess, with 21 files and many responsibilities designed to compensate for issues with Zap.  This mlog is 3 files and simply wraps the external logging engine to reduce dependency spread.  Most of it will get inlined.

This mlog will replace the mm-server version once I get some cycles to do that (it was a task I had upcoming when on server team anyway).